### PR TITLE
fix subclass weights error + add data description file

### PR DESCRIPTION
**Pull request by realirena**

### CODE REVIEW

File: R/plot-etiology-regression.R [hunk 1]
```diff
@@ -61,7 +61,7 @@
 #' @family visualization functions
 #' @export
 plot_etiology_regression <- function(DIR_NPLCM,stratum_bool,slice=1,plot_basis=FALSE,
-                                     truth=NULL,RES_NPLCM=NULL,do_plot=TRUE,return_metric=TRUE,
+                                     truth=NULL,RES_NPLCM=NULL,do_plot=TRUE,do_rug=FALSE, return_metric=TRUE,
```
Comment (zhenkewu) [inline R/plot-etiology-regression.R:64]: @realirena can we add do_rug in the the @param, so the function can recognize this argument.

File: code-in-progress/gg_plot_etiology_strat.R [hunk 2]
```diff
@@ -0,0 +1,124 @@
+
+##----------------------------
```
Comment (zhenkewu) [inline code-in-progress/gg_plot_etiology_strat.R:2]: @realirena this folder 'code-in-progress' will not make it into the final package. So can I assume code in this folder is not considered part of a real pull request?

Comment (realirena) [reply code-in-progress/gg_plot_etiology_strat.R]: Yes, removed it for now! In the future maybe we could add some ggplot features.

File: R/plot-etiology-regression.R (hunk 1 above)
Comment (realirena) [reply R/plot-etiology-regression.R:65]: Just updated so do_rug is in @param

**Comment by zhenkewu** [comment]
Just merged all!

**Comment by realirena** [comment]
I think it can be done, but I haven't tested it yet to make sure that it can work with the PERCH posterior samples. I will work on that and keep you updated.

**Pull request merged**